stubs/toml/toml/tz.pyi

```diff
@@ -0,0 +1,9 @@
+from datetime import datetime, timedelta, tzinfo
+from typing import Any
+
+class TomlTz(tzinfo):
+    def __init__(self, toml_offset: str) -> None: ...
+    def __deepcopy__(self, memo: Any) -> TomlTz: ...
```

Comment:
Even though it's unlikely to make any difference in practice, the return type is same as whatever the type of `self` is, and not `TomlTz` if you have a subclass of `TomlTz`.

```suggestion
    def __deepcopy__(self: Self, memo: Any) -> Self: ...
```

You will need to import `Self` from `_typeshed`.